Core/include/Acts/EventData/ParameterSet.hpp

```diff
@@ -77,32 +79,36 @@ using FullParameterSet = typename detail::full_parset::type;
  * @tparam params           parameter pack containing the (local) parameters
  * stored in this class
  */
-template <ParID_t... params>
+template <typename parameter_indices_t, parameter_indices_t... params>
```

Comment:
Should we default `parameter_indices_t`?

Reply:
I would not do that to avoid ambiguities. Whenever someone uses this, he should be forced to explicitely define what he wants to do.

Reply:
Fair enough.